tests/test.spec.js

```diff
@@ -0,0 +1,51 @@
+const expect = require('chai').expect;
+const yuGiOhApi = require('../YuGiOhApiV5/YuGiOhApi');
+
+describe('Yu-gi-oh card infos', function () {
+
+  this.timeout(10000);
+
+  beforeEach(() => {
+
+  });
+
+  afterEach(() => {
+
+  });
+
+  it('Verifies the exact name of a card querying by the exact name', async () => {
+    const card = await yuGiOhApi.cardInfo.getCardByName('Tornado Dragon');
+    expect(card.name).to.eq('Tornado Dragon');
+  });
+
+  it('Verifies the name of a card querying by ID', async () => {
+    const card = await yuGiOhApi.cardInfo.getCardByName(6983839);
+    expect(card.name).to.eq('Tornado Dragon');
+  });
+
+  it('Verifies the exact name of a card querying by name like Twin-Headed', async () => {
+    const cards = await yuGiOhApi.cardInfo.getCardsByNameLike('Twin-Headed');
+    expect(cards.find(card => card.name === 'Twin-Headed Thunder Dragon')).to.include({
+      name: 'Twin-Headed Thunder Dragon'
+    });
+  });
+
+  it('Verifies there are 8 cards with Attack more or equal than 8000', async () => {
```

Comment:
The string between single quotes should be:
Verifies there are 8 cards with Attack greater or equal to 8000